doc/sphinx/using/libraries/funind.rst

```diff
@@ -1,6 +1,15 @@
 Functional induction
 ====================
 
+.. note::
+
+   Most features of ``FunInd`` are also available in ``Equations``, which
+   is available in the Coq Platform. If ``Equations`` suits your needs,
+   we recommend you use it in new code instead of ``FunInd``. ``FunInd`` is
+   not deprecated and not planned for removal for now because some features
+   are not yet available in ``Equations`` (induction principle generation
+   like :cmd:`Function` does).
```

Comment:
this last precision seems unclear to me. A (imperfect) suggestion:
(:cmd:`Function` and :cmd:`Equations` generate incompatible induction principles).

Reply:
They are better imho in the sense that they follow the user's intent (wrt the order of patterns and the wildcards), but I am not sure it matters here. Does it?

Reply:
But yes incompatibility is the main (only?) reason for not deprecating Function yet. Porting a proof from the latter to the former needs 1) reordering cases in proofs 2) duplicating cases in proofs.